src/templates/static/home.jsx

```diff
@@ -353,20 +357,15 @@ const Home = () => {
                 </div>
             </div>
 
-            <div className='container gr-padding-30 gr-child'>
+            <div className='container gr-padding-30 gr-child academy-section'>
```

Comment:
is this class used anywhere?

Reply:
ah, we had some design changes and I forgot to remove this class